woocommerce/woocommerce
# add: updated payment gateways for 2023 Q3

### All Submissions:






### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/38237.

1. Ensure payment gateway display order is as per shown in [Google sheet](https://docs.google.com/spreadsheets/d/1cb4M2oa9C6H6I287JSM_oc_s_IP8VzipnTb1TMI8FbY/edit#gid=605747020). The ordering would be descending (i.e: The top has the highest priority).
2. Removed India from ppcp-gateway - others
3. Added India to ppcp-gateway - additional
4. Added Ecuador to mercado - others

(Although India was added to additional payment suggestions, additional payment suggestions only show up upon installation of WCPay and WCPay isn't supported for India, so there is no visible change.)






### How to test the changes in this Pull Request:



1. Start with a fresh install
1. Go to OBW
1. Choose Ecuador as store country
1. Complete OBW without installing any plugins in the business step.
1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com` > Untick "Show Suggestions" checkbox
1. Go to `WooCommerce -> Home` and choose Set up payments task.
1. Observe that `Mercado`, `PayPal Payments` are displayed in order.
1. Change store country to India
1. Observe that Paypal isn't showing up

![image](https://github.com/woocommerce/woocommerce/assets/27843274/bf6de74a-4c09-4c24-9eb5-2d73c842d78b)


Test the other countries and make sure the payment gateway display order is as shown in [Google sheet](https://docs.google.com/spreadsheets/d/1cb4M2oa9C6H6I287JSM_oc_s_IP8VzipnTb1TMI8FbY/edit#gid=0).





### Other information:




### FOR PR REVIEWER ONLY:
